airbyte-to-flow/src/interceptors/fix_document_schema.rs

```diff
@@ -187,7 +188,7 @@ pub fn fix_nonstandard_jsonschema_attributes(schema: &mut serde_json::Value) {
                 if f == "int32" || f == "int64" {
                     // Insert updates values
                     map.insert("format".to_string(), json!("integer"));
-                } else if f == "base64" {
+                } else if let Err(_) = serde_json::from_value::<Format>(serde_json::Value::String(f.to_string())) {
```

Comment:
nit: you could use `serde_json::from_str(f)` here

Reply:
oops, turns out these are not equivalent, since `from_str` would expect a quoted string, which we don't have here, this caused all `format` stanzas to be removed. reverting this nit: https://github.com/estuary/airbyte/pull/279